docs/how-to-guides/software-management/releases/isis-release-process.md

```diff
@@ -2,36 +2,59 @@
 
 !!! info "For releasing other Astro software, see the [general release process :octicons-arrow-right-16:](general-release-process.md)"
   
-The ISIS release process changes depending on the type of release: 
+The ISIS release process changes depending on the type of release.  Release Candidates (RCs) and Live Releases are only done for Major Releases (i.e, 8.x.x → 9.0.0). 
 
 - **RC** (Release Candidate)
-    - new feature releases are released first as RCs  
+    - Major releases are released first as RCs  
       (e.g. going from `8.0`->`8.1` or `8.5`->`9.0`)
 * **LR** (Live Release)
     - If no changes are suggested for the RC after a month,  
-      **do a LR from the RC**.  
+      **do a LR from the RC**.
+
+Subsequent Production and LTS releases (with no Major version change, i.e, 8.***2***.0 → 8.***3***.0, 9.0.***1*** → 9.0.***2***), have a more streamlined release process, with no RC/LR.
+
 * **LTS** (Long Term Support)
     - The previous major release is supported for a year in LTS.
-    - It only receives bug fixes, no feature adds. 
+    - Only receives bug fixes, no feature adds. 
+
+* **Prod** (Production)
+    - Quarterly release with (non-breaking) features and bugfixes.
+
+------
+
+### 1. **Prepare Feeder Branch/Merge Any Unmerged Changes**
+
+=== "RC"
+
+    - [ ] For initial preparation, work from your own branch and make PRs to the `dev` branch.  (Dev is the feeder branch for RCs.)
```

Comment:
This is true for the first RC, but subsequent RCs should likely be based off the previous RC branch